### ISSUE TITLE
`replaceStylesheets` question

### ISSUE THREAD

**Issue by XhmikosR** [bezoerb/inline-critical#266]
I was looking at this code @bezoerb:

https://github.com/bezoerb/inline-critical/blob/cfb890169e3d0af30bb57a907e8f470f3a6b697f/index.js#L93

`Array.isArray(o.replaceStylesheets) > 0` seems weird. Maybe we should expand the check to make it clearer?

Maybe:

```js
!Boolean(o.replaceStylesheets) && Array.isArray(o.replaceStylesheets) && o.replaceStylesheets.length> 0
```

Or

```js
!Boolean(o.replaceStylesheets) && o.replaceStylesheets.length> 0
```

PS. I hope I'm not overthinking this, it just seems weird to me :)
PPS. not sure if this affects the 4.x branch

**Comment by XhmikosR** [comment]
Or maybe you meant it to be:

```js
if (Array.isArray(o.replaceStylesheets) && links.length > 0) { 
```
?

**Comment by bezoerb** [comment]
Oh, this looks pretty shitty to me at the moment... i will need to check if there was anything useful in my brain when i wrote this ;)

**Comment by bezoerb** [comment]
Looks like i forgot the ` && o.replaceStylesheets.length` in between

**Comment by bezoerb** [comment]
Fixed in [`v5.3.1`](https://github.com/bezoerb/inline-critical/releases/tag/v5.3.1)
